test/solc/CommandLineParser.cpp

```diff
@@ -388,10 +388,10 @@ BOOST_AUTO_TEST_CASE(standard_json_mode_options)
 
 	CommandLineOptions expectedOptions;
 	expectedOptions.input.mode = InputMode::StandardJson;
-	expectedOptions.input.paths = {};
-	expectedOptions.input.standardJsonFile = "input.json";
+	expectedOptions.input.paths = {"input.json"};
 	expectedOptions.input.basePath = "/home/user/";
 	expectedOptions.input.allowedDirectories = {"/tmp", "/home", "project", "../contracts"};
+	expectedOptions.input.ignoreMissingFiles = true;
 	expectedOptions.output.dir = "/tmp/out";
```

Comment:
I'm not entirely sure, but wouldn't it be good to also add some tests here that test command line parameters that are usually used in different platforms, e.g. windows. Just to verify that the `CommandLineParser` parses e.g. windows specific paths correctly.

Reply:
True and I actually do have such tests but only for `CommandLineInterface` (in #11545). `CommandLineInterface` actually reads files and uses these paths. `CommandLineParser` only puts paths in the options struct so I think there's not much gain from repeating these tests for it. Especially given that these tests are pretty bulky and often need ifdefs for other platforms.

Actually, I could move the first commit from #11545 that contains them here but I'm not sure if this makes sense now that it's been reviewed. Originally I wanted it to be a separate PR but I already have too many PRs that depend on each other and I decided to avoid creating more.